annotations/security-schemes/README.md

```diff
@@ -11,14 +11,37 @@
 
 (see full [description](oauth-2-custom-settings.raml))
 
-### What's included?
+### What's included
 
 This directory contains the following RAML `AnnotationTypeDeclaration` fragment:
+
 - oauth-2-custom-settings.raml
 
 and the following example:
+
 - oauth-2-custom-settings-example.raml
 
 ### Which tools support this
 
-- [mulesoft/api-console (v5.0.x)](https://github.com/mulesoft/api-console)
+- [mulesoft/api-console (> v5.0.x)](https://github.com/mulesoft/api-console)
+
+## `pkce`
+
+### Purpose of the PKCE annotation
+
+Defines annotation that adds support for RFC 7636: Proof Key for Code Exchange.
+
+### Files related to PKCE support
+
+This directory contains the following RAML `AnnotationTypeDeclaration` fragment:
+
+- oauth-2-pkce.raml
+
+and the following example:
+
+- oauth-2-pkce-example.raml
+
+### API tools support PKCE
```

Comment:
should it be `Which tools support this?` like in the section above?

Reply:
It's linter, again. First is the question mark and the second is a duplicated title.

Reply:
I don't see why markdown would forbid duplicate subtitles as long as they are under different hierarchies, which the case here... again, I think there is an issue with the linter you're using.